### ISSUE TITLE
Concomitant use of tamoxifen and endoxifen in postmenopausal early breast cancer: prediction of plasma levels by physiologically-based pharmacokinetic modeling.

### ISSUE THREAD

**Issue by Stergiou** [Open-Systems-Pharmacology/OSP-based-publications#78]
https://www.ncbi.nlm.nih.gov/pubmed/24936398

Dickschen K1, Eissing T2, Mürdter T3, Schwab M4, Willmann S5, Hempel G6.

Abstract

PURPOSE: 

To overcome cytochrome P450 2D6 (CYP2D6) mediated tamoxifen resistance in postmenopausal early breast cancer, CYP2D6 phenotype-adjusted tamoxifen dosing in patients with impaired CYP2D6 metabolism and/or the application of endoxifen, the most potent tamoxifen metabolite, are alternative treatment options. To elucidate both strategies comprehensively we used a physiologically-based pharmacokinetic (PBPK) modeling approach.

METHODS: 

Firstly simulation of increasing tamoxifen dosages was performed by a virtual clinical trial including populations of CYP2D6 poor (PM), intermediate (IM) and extensive metabolizers (EM) (N = 8,000). Secondly we performed PBPK-simulations under consideration of tamoxifen use plus concomitant increasing dosages of endoxifen (N = 7,000).

RESULTS: 

Our virtual study demonstrates that dose escalation of tamoxifen in IMs resulted in endoxifen steady-state plasma concentrations similar to CYP2D6 EMs whereas PMs did not reach EM endoxifen levels. Steady-state plasma concentrations of tamoxifen, N-desmethyl-tamoxifen, 4-hydroxy-tamoxifen and endoxifen were similar in CYP2D6 IMs and PMs versus EMs using once daily dosing of 20 mg tamoxifen and concomitant CYP2D6 phenotype-adjusted endoxifen dosing in IMs and PMs (1 mg/d and 3 mg/d, respectively).

CONCLUSION: 

In conclusion, we suggest that co-administration of endoxifen in tamoxifen treated early breast cancer women with impaired CYP2D6 metabolism is a promising alternative to reach plasma concentrations comparable to CYP2D6 EM patients.

KEYWORDS: 

CYP2D6 genotype; PBPK modeling; Postmenopausal breast cancer; Tamoxifen-endoxifen-combination

PMID: 24936398 PMCID: PMC4058004 DOI: 10.1186/2193-1801-3-285 

Free PMC Article
